decorator.rb

```diff
@@ -0,0 +1,28 @@
+require './nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
```

Comment:
- [ ] Good job so far 👍 Kindly add the `attr_accessor :nameable` to your `Decorator` class 💯